### ISSUE TITLE
Unsequenced modification and access to 't'

### ISSUE THREAD

**Issue by thinium** [arturoc/ofxEasing#1]
Hi, thanks for the nice work! 

According to the following thread it seems that (t/=d) \* t is an undefined behavior. 

http://stackoverflow.com/questions/18729323/unsequenced-modification-and-access-to-parameter

While I have no problem using this addon and can disable the warning easily, I just want to make sure you are aware of this issue.

**Comment by arturoc** [comment]
yes, i know, the code for the equations was ported by @jesusgollonet years ago and has been used in lots of different addons and projects. although compilers signal those assignments as a warning it works fine in every platform and compiler we've tried until now.
